src/common/playleaderboard/leaderBoard.css

```diff
@@ -0,0 +1,18 @@
+.table-cell {
```

Comment:
The css class names are very generic.. may conflict.

Reply:
changed.